### ISSUE TITLE
VSCode 1.21: command prettifySymbolsMode.togglePrettySymbols not found

### ISSUE THREAD

**Issue by Narfinger** [siegebell/vsc-prettify-symbols-mode#33]
With the current version of vscode, this extension (as installed from the marketplace) does not seem to work anymore. At least the commands to toggle it, enable or disable it show the above error

**Comment by chi-feng** [comment]
I'm getting the same error for all 4 of the contributed commands.

**Comment by boughia** [comment]
Encountered the same issue. I tried debugging the extension and got the following error:

![ptm](https://user-images.githubusercontent.com/16523634/37283090-b7b50a24-2631-11e8-85c9-2a31f545302e.PNG)

Then I checked my vs code directory and under resources/app there was no node_modules folder. So as a workaround I installed the vscode-textmate node package into resources/app, and then the extension works properly.
